easybuild/easyconfigs/b/Bader/Bader-1.04-GCC-11.2.0.eb

```
@@ -0,0 +1,28 @@
+easyblock = 'MakeCp'
+
+name = 'Bader'
+version = '1.04'
+
+homepage = 'https://theory.cm.utexas.edu/henkelman/code/bader/'
+description = "Bader Charge Analysis"
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
+
+source_urls = ['http://theory.cm.utexas.edu/henkelman/code/bader/download/v%(version)s']
+sources = [{'download_filename': 'bader.tar.gz', 'filename': SOURCE_TAR_GZ}]
+checksums = ['f20a0a021157d911bea06666911763b737c4ff38b39e793b8560f940fe391b8e']
+
+buildopts = ' -f makefile.lnx_ifort FC=$FC FFLAGS="$FFLAGS" '
+
+parallel = 1
+
+files_to_copy = [
+    (['bader'], 'bin'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/bader'],
+    'dirs': [],
+}
+
+moduleclass = 'phys'
```

In the other easyconfigs for Bader we are using `moduleclass = 'chem'`. In my opinion `phys` makes also sense, but for consistency I would prefer to keep `chem`.

```suggestion
moduleclass = 'chem'
```

---

Since Bader deals with atoms at the physical level and really has nothing to do with chemistry as such I still prefer phys here.
It's main targets are VASP and Gaussian, i.e. material physics not chemistry

---

Thanks for the explanations. :+1: